### ISSUE TITLE
Implement fake property for karaoke text

### ISSUE THREAD

**Issue by andy840119** [osu-Karaoke/osu-karaoke-framework#18]
Implement fake property for karaoke text.
1. Some missing property not apply, see todo in `DrawableLyricLine`
2. Implement `ILyricTextureSample` interface

**Comment by andy840119** [comment]
`ILyricTextureSample` interface will looks like : 
```Csharp
interface ILyricTextureSample
{
  Texture CreateTexture(int width,int height,Orientation orientation)
}
```
container three inherit class
```csharp
class SolidTextureSample : ILyricTextureSample
{
 Color4 SolidColor{get;set;}
}

class MixedextureSample : ILyricTextureSample
{
 IReadOnlyDictionary<int,Color4> Colors{get;set;}

 enum MixedType
 {
   Gradient,
   MilleFeuille
 }

}
```

**Comment by andy840119** [comment]
Implemented in #19
